engine/src/org/pentaho/di/cluster/SlaveServer.java

```diff
@@ -266,13 +267,13 @@ public String getXML() {
     return xml.toString();
   }
 
-  public Object clone() {
+  public synchronized Object clone() {
```

Comment:
This makes no sense either. The substantive "work" done here reading instance variables is done in replaceMeta. Yet, you synchronize here, and synchronize replaceMeta. This is not rational.